src/ModelContextProtocol/ModelContextProtocol.csproj

```diff
@@ -4,8 +4,8 @@
     <TargetFrameworks>net9.0;net8.0;netstandard2.0</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IsPackable>true</IsPackable>
-    <PackageId>ModelContextProtocol</PackageId>
-    <Description>.NET SDK for the Model Context Protocol (MCP)</Description>
+    <PackageId>ModelContextProtocol.Core</PackageId>
```

Comment:
Sorry, when I said minimal changes, I did not mean to not move files around. In addition to renaming this PackageId to ModelContextProtocol.Core, we should rename this file to ModelContextProtocol.Core.csproj and put it in the src/ModelContextProtocol.Core directory. After doing that, the src/ModelContextProtocol.Hosting folder should be renamed to src/ModelContextProtocol and ModelContextProtocol.Hosting.csproj should be renamed to ModelContextProtocol.csrpoj. It's important to move directories in this order because we don't want to clobber anything.